src/openforms/config/templatetags/privacy_policy.py

```diff
@@ -7,9 +7,10 @@
 register = template.Library()
 
 
-@register.simple_tag()
-def privacy_policy():
-    conf = GlobalConfiguration.get_solo()
+@register.simple_tag(takes_context=True)
+def privacy_policy(context):
+
+    conf = context.get("global_configuration") or GlobalConfiguration.get_solo()
```

Comment:
small optimization where we can avoid a cache hit and potentially a DB hit

Reply:
My knee-jerk reaction to this circular way of rendering  a link in our own property was why? But setting the correct `rel` attribute on the anchor is *that* important.  :+1:  And I guess anchors aren't allowed in the sandbox?